src/cpu/gemm/gemm.cpp

```diff
@@ -192,6 +196,25 @@ dnnl_status_t gemm_s8x8s32(const char *transa, const char *transb,
                 B, LDB, bo, beta, C, LDC, co, false);
 #endif
 
+#ifdef DNNL_PPC64
+#ifdef __MMA__
```

Comment:
QQ: why is this needed?

Reply:
From what I understand, `DNNL_PPC64` is defined whenever we are running on any "Power" box (Power7, Power8, Power9, Power10).  However, the "MMA engine" is new with Power10, so we must also restrict access by using the environment variable `__MMA__` which is only defined for boxes which contain that feature.